pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -79,6 +79,9 @@ type AzVolumeStatus struct {
 	//Current phase of the underlying PV
 	//+optional
 	Phase AzVolumePhase `json:"phase,omitempty"`
+	//Error occured during creation/deletion of volume
+	//+optional
+	AzVolumeError *AzError `json:"azVolumeError,omitempty"`
```

Comment:
One last thought: we should simply name these fields `Error` to avoid the stutter that Go general frowns on (i.e. `AzVolume.AzVolumeError`. The fact that it's an error specific to the `AzVolume` is implied by the struct type, and does not need to be repeated in the field name.

Reply:
Makes sense.